### ISSUE TITLE
Scrolling when the focus is in the commit area scrolls both the commit scrollbar and the global scrollbar.

### ISSUE THREAD

**Issue by Ultra-Instinct-05** [sublimehq/sublime_merge#1326]
**Version info**

 - OS: Windows 11
 - Build: 2062 (Dev)

**Description**

When the focus is in the commit message text control and it's lengthy enough to have a scrollbar (And there are sufficient changes in the form of staged + unstaged files) to have a global scrollbar as well, then focusing the commit message and scrolling scrolls both the commit scrollbar & the global scrollbar.

**Steps to reproduce**

1. Have a lengthy commit message to get a scrollbar in the commit area.
2. Have enough staged or unstaged changes to get a scrollbar.
3. Focus the commit message & start scrolling.

**Expected behavior**

Only the commit message should scroll.

**Actual behavior**

Both the commit message and the global scrollbar (that encompasses commit text control and diff hunks) scroll.

**Screenshots**

![1556789999](https://user-images.githubusercontent.com/34304644/142137511-38067c74-c263-487d-b03c-b230736f159e.gif)

**Comment by srbs** [comment]
On my Mac (10.14; build 2062) this behavior is not seen, instead only one scrolls at a time. i.e. once at the bottom of the commit view, the global view starts scrolling & vice versa. I think this is the intended behavior.

**Comment by Ultra-Instinct-05** [comment]
> On my Mac (10.14; build 2062) this behavior is not seen,

Might be OS specific then.

>  I think this is the intended behavior.

I don't see this as an intended behavior. When the focus is in the commit message, I don't expect the global view to scroll, only the commit scrollbar to scroll. When the focus is in the global view, I don't expect the commit scrollbar to scroll, only the global scrollbar to scroll.

Having both scroll at the same time results in a janky experience.

**Comment by srbs** [comment]
> Having both scroll at the same time results in a janky experience.

Agreed, I would prefer only what is hovered over (not focused) to be the component to scroll.
